.eslintrc

```diff
@@ -1,19 +1,37 @@
 {
```

Comment:
[OOS/Future] Would be cool to switch this over to the eslint JS config style

- https://eslint.org/docs/user-guide/configuring